modules/optimeraBidAdapter.md

```diff
@@ -0,0 +1,40 @@
+# Overview
+
+```
+Module Name: Optimera Bidder Adapter
+Module Type: Bidder Adapter
+Maintainer: kcandiotti@optimera.nyc
+```
+
+# Description
+
+Module that adds ad placement visibility scores for DFP.
+
+# Test Parameters
+```
+    var adUnits = [{
+                code: 'div-1',
+                sizes: [[300, 250], [300,600]],
+                bids: [
+                    {
+                        bidder: 'optimera',
+                        params: {
+                            custom:{
```

Comment:
why do you have the `custom` wrapper here? seems unnecessary.

Reply:
Ok, we removed the custom wrapper and updated the adapter and tests to reflect the change.